api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/dashboards_controller.rb

```diff
@@ -95,6 +95,7 @@ def dashboard_params
         whitelisted[:settings] = params[:dashboard][:metadata] || {}
       end
       .except(:metadata)
+      .merge(owner_type: "User", owner_id: current_user.id)
```

Comment:
This will cause backward compatibility issues with some customer projects where the owner is an Org.
They've extended the `dashboards` method.
I can share the code customisation with you as I believe it's quite flexible.